cmake/googletest_download.txt.in

```diff
@@ -10,13 +10,9 @@ project(googletest-download NONE)
 
 include(ExternalProject)
 
-set(CMAKE_CXX_STANDARD 11)
-set(CMAKE_CXX_EXTENSIONS OFF)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
Moved these variables to the googletest.cmake file which is the one which calls configure_file() on this one (and the src one below). For some reason the value of these vars was not being used in the call to ExternalProject_Add(). So actually since they were added they have never been working.